Fix pagingEnabled not working.

chickdan opened the pull request:
If `collectionViewPaginatedScroll` was true it set `collectionView.isPagingEnabled` to false and it was never set to true anywhere. So setting `paginationEnabled` inside of GLTableCollectionViewController did nothing.

giulio92 commented:
Hello @chickdan,

Could you explain better the purpose of this `PR`? I think that the current code follows the behavior I thought in the beginning, do you see any problems with this?

Thank you

chickdan commented:
Sure, so first when I was playing with the demo I didn't notice a change in behavior when I changed the boolean `paginationEnabled` (as per the directions in the Readme). So I took a look in GLCollectionTableViewCell and noticed that you had:

 ```
if collectionViewPaginatedScroll == true {
    collectionView.isPagingEnabled = false 
}
```

Which is the opposite of the desired behavior, if the boolean is true you want it to paginate. 

Second, because there is no else statement `isPagingEnabled` is never set to the opposite value. So in the context of your current code, isPagingEnabled is false by default when the collectionView is created and therefore never set to true because of the above if statement. 

So with this PR I simply switched the false to true and added an else so that the value can be switched.

Edit: I was pretty tired when I made the change and after thinking about it, there shouldn't even be a conditional. It should be `collectionView.isPagingEnabled = collectionViewPaginatedScroll`

giulio92 commented:
Hello @chickdan,

Thank you really much for the explanation 😊, I will try your changes and see how the new behavior works 😊

Thank you really much for your help!
